parts_list/master_parts_list_raw.csv

```diff
@@ -134,10 +134,11 @@ Metal Hacksaw or Bandsaw,D4,,,,,1,,,,"Rocker-Bogie, Head Assembly, Wheel Assembl
 Vice clamp or C Clamps,D5,,,,,1,,,,"Wheel Assembly, Differential Pivot"
 5/16 Wrench,D1,,,,,1,,,,"Rocker-Bogie, Mechanical Integration, Wheel Assembly, Differential Pivot"
 Vice or V-Clamp,D8,,,,,1,,,,"Head Assembly, Differential Pivot"
-YellowJacket 223 rpm drive motor,E49,5202-0002-0027,GoBilda,https://www.gobilda.com/5202-series-yellow-jacket-planetary-gear-motor-26-9-1-ratio-223-rpm-3-3-5v-encoder/,1,6,6,$39.99,$239.94,Wheel Assembly
-M4 x 0.7mm 6mm screws,B15,2802-0004-0006,GoBilda,https://www.gobilda.com/2802-series-zinc-plated-steel-button-head-screw-m4-x-0-7mm-6mm-length-25-pack/,25,48,2,$1.79,$3.58,Wheel Assembly
-GoBilda 2 hole U-Channel,S45,1120-0002-0072,GoBilda,https://www.gobilda.com/1120-series-u-channel-2-hole-72mm-length/,1,6,6,$2.79,$33.48,Wheel Assembly
-GoBilda 23mm threaded plate,S46,2803-0023-0023,GoBilda,https://www.gobilda.com/2803-series-stainless-steel-threaded-plate-23-23/,1,12,12,$2.49,$29.88,Wheel Assembly
-2-post 43-32mm drive motor clamping mount,S47,1401-0043-0032,GoBilda,https://www.gobilda.com/1401-series-2-side-2-post-clamping-mount-43mm-width-32mm-bore/,1,6,6,$5.99,$35.94,Wheel Assembly
-GoBilda Pattern Adaptor,S48,1206-0016-0004,GoBilda,https://www.gobilda.com/1206-series-pattern-adaptor-16-4/,1,12,12,$4.99,$59.88,Wheel Assembly
-6mm Bore Clamping D-Hub 0.770in,S49,545618,ServoCity,https://www.servocity.com/6mm-bore-clamping-d-hub-tapped-0-770-pattern/,1,6,6,$7.99,$47.94,Wheel Assembly
+YellowJacket 223 rpm drive motor,E46,5202-0002-0027,GoBilda,https://www.gobilda.com/5202-series-yellow-jacket-planetary-gear-motor-26-9-1-ratio-223-rpm-3-3-5v-encoder/,1,6,6,$39.99 ,$239.94 ,Wheel Assembly
+M4 x 0.7mm 6mm screws,B15,2802-0004-0006,GoBilda,https://www.gobilda.com/2802-series-zinc-plated-steel-button-head-screw-m4-x-0-7mm-6mm-length-25-pack/,25,48,2,$1.79 ,$3.58 ,Wheel Assembly
+GoBilda 2 hole U-Channel,S45,1120-0002-0072,GoBilda,https://www.gobilda.com/1120-series-u-channel-2-hole-72mm-length/,1,6,6,$2.79 ,$33.48 ,Wheel Assembly
+GoBilda 23mm threaded plate,S46,2803-0023-0023,GoBilda,https://www.gobilda.com/2803-series-stainless-steel-threaded-plate-23-23/,1,12,12,$2.49 ,$29.88 ,Wheel Assembly
+2-post 43-32mm drive motor clamping mount,S47,1401-0043-0032,GoBilda,https://www.gobilda.com/1401-series-2-side-2-post-clamping-mount-43mm-width-32mm-bore/,1,6,6,$5.99 ,$35.94 ,Wheel Assembly
+GoBilda Pattern Adaptor,S48,1206-0016-0004,GoBilda,https://www.gobilda.com/1206-series-pattern-adaptor-16-4/,1,12,12,$4.99 ,$59.88 ,Wheel Assembly
+6mm Bore Clamping D-Hub 0.770in,S49,545618,ServoCity,https://www.servocity.com/6mm-bore-clamping-d-hub-tapped-0-770-pattern/,1,6,6,$7.99 ,$47.94 ,Wheel Assembly
+9 x 12 Bottom Plate,S51,,Sculpteo,https://www.sculpteo.com/,1,1,1,$11.43 ,$11.43 ,Body
```

Comment:
Somehow comments got introduced here but should be OK

Reply:
fixed